versions.json

```diff
@@ -1 +1 @@
-["v3.29.4", "v4.10.2"]
+[]
```

Comment:
@fanciulli was this your solution to this error?
`[ERROR] Error: The docs folder does not exist for version "v3.29.4". A docs folder is expected to be found at versioned_docs/version-v3.29.4.
`